doc/source/reference/c-api.array.rst

```diff
@@ -1921,6 +1921,7 @@ Calculation
 
     Equivalent to :meth:`ndarray.max` (*self*, *axis*). Return the largest
     element of *self* along the given *axis*.
+    If it return only 1 element, return a numpy scalar, not an ndarray.
```

Comment:
`If the result is a single element, then a numpy scalar is returned instead of a 0-d array.`